src/element/newElement.test.ts

```diff
@@ -15,79 +16,310 @@ const assertCloneObjects = (source: any, clone: any) => {
   }
 };
 
-it("clones arrow element", () => {
-  const element = API.createElement({
-    type: "arrow",
-    x: 0,
-    y: 0,
-    strokeColor: "#000000",
-    backgroundColor: "transparent",
-    fillStyle: "hachure",
-    strokeWidth: 1,
-    strokeStyle: "solid",
-    roundness: { type: ROUNDNESS.PROPORTIONAL_RADIUS },
-    roughness: 1,
-    opacity: 100,
-  });
+describe("duplicating single elements", () => {
+  it("clones arrow element", () => {
+    const element = API.createElement({
+      type: "arrow",
+      x: 0,
+      y: 0,
+      strokeColor: "#000000",
+      backgroundColor: "transparent",
+      fillStyle: "hachure",
+      strokeWidth: 1,
+      strokeStyle: "solid",
+      roundness: { type: ROUNDNESS.PROPORTIONAL_RADIUS },
+      roughness: 1,
+      opacity: 100,
+    });
+
+    // @ts-ignore
+    element.__proto__ = { hello: "world" };
```

Comment:
lets add a comment about this

Reply:
not my test :) and I have no clue why we're testing for it

Reply:
btw, you should be reviewing with whitespace disabled so it's clear what's actually a change, and what's whitespace noop